daft/dataframe/dataframe.py

```diff
@@ -1120,6 +1120,36 @@ def __len__(self):
         )
         raise RuntimeError(message)
 
+    @DataframePublicAPI
```

Comment:
Let's not implement `fillnan` and `fillnull` on a DataFrame level?

You're correct that this `Only works if the columns have the same dtype.` - because of this, these functions will likely not work on most Dataframes.

This is almost never the case when working with Daft Dataframes because unlike a framework such as Pandas/Numpy, we deal with much more than just numerical data!